cpu/native/periph/flashpage.c

```diff
@@ -27,6 +27,7 @@
 #define ENABLE_DEBUG (0)
```

Comment:
```suggestion
#define ENABLE_DEBUG 0
```

Care to also fix this?

Reply:
done